### PR TITLE
[zh] translate blog "Don't Panic: Kubernetes and Docker"

### LINKED ISSUE #26502: [zh] translate blog "Don't Panic: Kubernetes and Docker"
**This is a Feature Request**
translate /blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

**What would you like to be added**
content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

/assign
/language zh

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

close #26502

### CODE REVIEW

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+slug: dont-panic-kubernetes-and-docker
+---
+<!-- 
+layout: blog
+title: "Don't Panic: Kubernetes and Docker"
+date: 2020-12-02
+slug: dont-panic-kubernetes-and-docker
+-->
+
+**Authors:** Jorge Castro, Duffie Cooley, Kat Cosgrove, Justin Garrison, Noah Kantrowitz, Bob Killen, Rey Lejano, Dan “POP” Papandrea, Jeffrey Sica, Davanum “Dims” Srinivas
```
Comment (sftim) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:14]: if you want to, it's OK to localize “Authors” - see https://kubernetes.io/zh/blog/2020/12/18/kubernetes-1.20-pod-impersonation-short-lived-volumes-in-csi/ for an example.

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+One thing to note: If you are relying on the underlying docker socket
+(`/var/run/docker.sock`) as part of a workflow within your cluster today, moving
+to a different runtime will break your ability to use it. This pattern is often
+called Docker in Docker. There are lots of options out there for this specific
+use case including things like
+[kaniko](https://github.com/GoogleContainerTools/kaniko),
+[img](https://github.com/genuinetools/img), and
+[buildah](https://github.com/containers/buildah). 
+-->
+要注意一点：如果你依赖底层的 docker 套接字(`/var/run/docker.sock`)，作为你集群中工作流的一部分，
```
Comment (sftim) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:148]: tiny nit:
I would write Docker in title case here
```suggestion
要注意一点：如果你依赖底层的 Docker 套接字(`/var/run/docker.sock`)，作为你集群中工作流的一部分，
```
even though in the original it is lower case.

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+Kubernetes 有很多变化中的功能，没有人是100%的专家。
+我们鼓励你提出任何问题，无论水平高低、问题难易。
+我们的目标是确保所有人都能在即将到来的改变中获得足够的了解。
+我们希望这已经回答了你的大部分问题，并缓解了一些焦虑。❤️
+
+<!-- 
+Looking for more answers? Check out our accompanying [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/).
+-->
+还在寻求更多答案吗？请参考我们附带的
+[弃用 Dockershim 的常见问题](/blog/2020/12/02/dockershim-faq/).
```
Comment (sftim) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:210]: There's already a localized version available, so you could:
```suggestion
[弃用 Dockershim 的常见问题](/zh/blog/2020/12/02/dockershim-faq/).
```

**Review by sftim — COMMENTED** [review]
A few small suggestions - I hope these are useful.

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md (hunk 1 above)
Comment (zhiguo-lu) [reply content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:14]: got it, fixed

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md (hunk 3 above)
Comment (zhiguo-lu) [reply content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:210]: wow, this piece is also my translation , corrected.

**Comment by zhiguo-lu** [comment]
> A few small suggestions - I hope these are useful.

sure, it helps me. Thanks a lot.

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+tech stack, and one part of it is a thing called “containerd,” which is a
+high-level container runtime by itself. Docker is cool and useful because it has
+a lot of UX enhancements that make it really easy for humans to interact with
+while we’re doing development work, but those UX enhancements aren’t necessary
+for Kubernetes, because it isn’t a human. 
+-->
+你看，我们称之为 “Docker” 的物件实际上并不是一个物件——它是一个完整的技术堆栈，
+它其中一个叫做 “containerd” 的部件本身，才是一个高级容器运行时。
+Docker 既酷炫又实用，因为它有很多用户体验增强功能，这我们做开发工作时简化了操作，
+Kubernetes 用不着用户增强，毕竟它并非人类。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:110]:
```suggestion
Kubernetes 用不到这些增强的用户体验，毕竟它并非人类。
```

**Review by tengqm — COMMENTED** [review]
/approve

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+<!-- 
+If you’re an end-user of Kubernetes, not a whole lot will be changing for you.
+This doesn’t mean the death of Docker, and it doesn’t mean you can’t, or
+shouldn’t, use Docker as a development tool anymore. Docker is still a useful
+tool for building containers, and the images that result from running `docker
+build` can still run in your Kubernetes cluster. 
+--> 
+如果你是 Kubernetes 的终端用户，这对你不会有太大影响。
+这事并不意味着 Dockder 已死、也不意味着你不能或不该继续把 Docker 用作开发工具。
+Docker 仍然是构建容器的利器，使用命令 `docker build` 构建的镜像在 Kubernetes 集群中仍然好用。
```
Comment (ydFu) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:49]:
```suggestion
Docker 仍然是构建容器的利器，使用命令 `docker build` 构建的镜像在 Kubernetes 集群中仍然可以运行。
```

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+If you’re using a managed Kubernetes service like GKE, EKS, or AKS (which [defaults to containerd](https://github.com/Azure/AKS/releases/tag/2020-11-16)) you will need to
+make sure your worker nodes are using a supported container runtime before
+Docker support is removed in a future version of Kubernetes. If you have node
+customizations you may need to update them based on your environment and runtime
+requirements. Please work with your service provider to ensure proper upgrade
+testing and planning. 
+-->
+如果你正在使用 GKE、EKS、或 AKS 
+([默认使用 containerd](https://github.com/Azure/AKS/releases/tag/2020-11-16))  
+这类托管 Kubernetes 服务，你需要在 Kubernetes 后续版本移除 Docker 支持之前，
```
Comment (ydFu) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:61]:
```suggestion
这类托管 Kubernetes 服务，你需要在 Kubernetes 后续版本移除对 Docker 支持之前，
```

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+Kubernetes 有很多变化中的功能，没有人是100%的专家。
+我们鼓励你提出任何问题，无论水平高低、问题难易。
+我们的目标是确保所有人都能在即将到来的改变中获得足够的了解。
+我们希望这已经回答了你的大部分问题，并缓解了一些焦虑。❤️
+
+<!-- 
+Looking for more answers? Check out our accompanying [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/).
+-->
+还在寻求更多答案吗？请参考我们附带的
+[弃用 Dockershim 的常见问题](/zh/blog/2020/12/02/dockershim-faq/).
```
Comment (ydFu) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:210]:
```suggestion
[弃用 Dockershim 的常见问题](/zh/blog/2020/12/02/dockershim-faq/)。
```

File: content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+-->
+所以，改变已经发生。
+它确实带来了一些问题，但这不是一个灾难，总的说来，这还是一件好事。
+根据你操作 Kubernetes 的方式的不同，这可能对你不构成任何问题，或者也只是意味着一点点的工作量。
+从一个长远的角度看，它使得事情更简单。
+如果你还在困惑，也没问题——这里还有很多事情；
+Kubernetes 有很多变化中的功能，没有人是100%的专家。
+我们鼓励你提出任何问题，无论水平高低、问题难易。
+我们的目标是确保所有人都能在即将到来的改变中获得足够的了解。
+我们希望这已经回答了你的大部分问题，并缓解了一些焦虑。❤️
```
Comment (ydFu) [inline content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:204]:
```suggestion
我们希望这已经回答了你的大部分问题，并缓解了一些焦虑！❤️
```

**Review by ydFu — COMMENTED** [review]
Good job ! Thank you for your contribution and a little suggestion for you.

**Comment by zhiguo-lu** [comment]
> Good job ! Thank you for your contribution and a little suggestion for you.

review的非常详细，谢谢你的建议。

**Comment by ydFu** [comment]
/lgtm
thank.

**Pull request merged**